lib/const.rb

```diff
@@ -1,6 +1,6 @@
 # Defines common constants used in different parts of crew
 
-CREW_VERSION = '1.20.0'
+CREW_VERSION = '1.20.1'
```

Comment:
We cannot reuse `1.20.1`.  Bump to `1.20.2`?

Reply:
Done